client/my-sites/domains/paths.js

```diff
@@ -61,7 +61,7 @@ export function domainAddNew( siteName, searchTerm ) {
 }
 
 export function domainAddEmailUpsell( siteName, domainName ) {
-	return `/domains/add/${ siteName }/email/${ domainName }`;
+	return `/domains/add/${ domainName }/email/${ siteName }`;
```

Comment:
Also wanted to point out that I got the order of variables in template literal wrong in the original PR 🙈

Fixed it here.

Reply:
Thanks for catching this! 😳